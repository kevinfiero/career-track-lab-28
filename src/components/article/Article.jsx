import React from 'react';
import PropTypes from 'prop-types';

const Article = ({ article }) => (
  <h1>{article}</h1>
);

Article.propTypes = {
  article: PropTypes.string.isRequired
};

export default Article;
