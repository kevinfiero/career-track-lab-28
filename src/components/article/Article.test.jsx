import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './Article';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('renders Article', () => {
    const { asFragment } = render(<Article  
      //bgColor="#FF0000" 
      //fgColor="#0000FF" 
      //text="Hello!" 
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
