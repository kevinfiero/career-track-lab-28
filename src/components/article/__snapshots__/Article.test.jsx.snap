// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <h1>
    Extra! Extra!
  </h1>
</DocumentFragment>
`;
